cmd/argocd/commands/app.go

```diff
@@ -2154,6 +2155,33 @@ func NewApplicationEditCommand(clientOpts *argocdclient.ClientOptions) *cobra.Co
 	return command
 }
 
+func NewApplicationListOrphanedResourceCommand(clientOpts *argocdclient.ClientOptions) *cobra.Command {
```

Comment:
I think listing orphaned resources is a too specific operation to create separate command for it. If we go this way we might end up having too many specific commands.


Instead, let's create `argocd app resources` command that prints information about application resources and provides various filters. 

```bash
argocd app resources # list all resources orphaned and non-orphaned.
```

```bash
argocd app resources orphaned=true # lists  only orphaned resources
```

```bash
argocd app resources orphaned=false # lists only non-orphaned resources
```